### ISSUE TITLE
Unable to run "make sysprep" in High Sierra due to SIP

### ISSUE THREAD

**Issue by clong** [facebook/osquery#5116]
# Bug report

### What operating system and version are you using?
MacOS 10.13.6

### What version of osquery are you using?
3.3.0

### What steps did you take to reproduce the issue?
git clone https://github.com/facebook/osquery.git
cd osquery
make sysprep

### What did you expect to see?
The step complete successfully

### What did you see instead?
```
$ make sysprep
-- Warning! The only Apple OS supported for building is 10.12
-- Note: Installing and running osquery is supported on versions 10.9+
[+] found darwin provision script: ~/osquery/tools/provision/darwin.sh
[+] requesting sudo: /usr/bin/gem install --no-ri --no-rdoc fpm
Password:
ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions for the /usr/bin directory.
make: *** [sysprep] Error 1
```

I think this can be solved by changing the fpm gem to install to `/usr/local/bin` rather than `/usr/bin` in `/tools/provision/darwin.sh`. I'll put up a PR for this shortly.

**Comment by clong** [comment]
I think this can be closed now that #5117 is merged, unless you guys are waiting for a new release to test.

**Comment by guliashvili** [comment]
thanks for working on that
